pkg/db/run.go

```diff
@@ -403,13 +403,23 @@ type RunFindQuery struct {
 	//
 	// If it is nil or empty, it means "match any".
 	Status []KnitRunStatus
+
+	// match if run's updated time is equal or later than this UpdatedSince.
+	UpdatedSince *time.Time
+
+	// match if run's updated time is equal or earlier than this UpdatedUntil.
+	UpdatedUntil *time.Time
 }
 
 func (rfq RunFindQuery) Equal(other RunFindQuery) bool {
 	return cmp.SliceContentEq(rfq.PlanId, other.PlanId) &&
 		cmp.SliceContentEq(rfq.InputKnitId, other.InputKnitId) &&
 		cmp.SliceContentEq(rfq.OutputKnitId, other.OutputKnitId) &&
-		cmp.SliceContentEq(rfq.Status, other.Status)
+		cmp.SliceContentEq(rfq.Status, other.Status) &&
+		((rfq.UpdatedSince == nil && other.UpdatedSince == nil) ||
+			(rfq.UpdatedSince != nil && other.UpdatedSince != nil && *rfq.UpdatedSince == *other.UpdatedSince)) &&
```

Comment:
`time.Time` can not be compaird with `==`.

Use [`time.Time.Equal`](https://pkg.go.dev/time#Time.Equal).